lib/cdo/firehose.rb

```diff
@@ -61,6 +61,10 @@ def put_record(data)
     # backing off and retrying.
     # See http://docs.aws.amazon.com/sdkforruby/api/Aws/Firehose/Client.html#put_record-instance_method.
     Honeybadger.notify(e)
+  rescue StandardError => error
```

Comment:
Two suggestions:

1. This clause is strictly a superset of, and has identical behavior to, the one above that rescues ServiceError.  I think we could just replace the above clause with this one.
2. You can just write `rescue => error` which is equivalent to `rescue StandardError => error`.  See [Thoughbot](https://robots.thoughtbot.com/rescue-standarderror-not-exception).